contracts/mocks/extensions/revealable/RMRKMultiAssetRevealableMock.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.21;
+
+import "../../../RMRK/extension/revealable/RMRKRevealable.sol";
+import "../../RMRKMultiAssetMock.sol";
+
+contract RMRKMultiAssetRevealableMock is RMRKMultiAssetMock, RMRKRevealable {
+    function supportsInterface(
+        bytes4 interfaceId
+    )
+        public
+        view
+        virtual
+        override(RMRKMultiAsset, RMRKRevealable)
+        returns (bool)
+    {
+        return
+            RMRKMultiAsset.supportsInterface(interfaceId) ||
+            RMRKRevealable.supportsInterface(interfaceId);
+    }
+
+    /**
+     * @inheritdoc IRMRKRevealable
+     */
+    function setRevealer(address revealer) external {
```

Comment:
hm so anyone can call this?

Reply:
Glad you catched it!
This is the case, but it's ok since this is a mock just for testing,